templates/Program/program_description.html.twig

```diff
@@ -50,13 +51,32 @@
             {% endif %}
           </span>
       </div>
-      <div id="credits" class="text-break">
-        {% if program.credits %}
-          {{ program.credits|nl2br }}
-        {% else %}
+      {% if program.credits %}
+        <div id="credits" class="text-break">
```

Comment:
why are the `credits` div and the byline div duplicated?

Reply:
Added an extra if statement to remove the duplicate byline div

Reply:
this `div` is still duplicated 
https://github.com/danielzhang130/Catroweb/blob/ac868e517b2db86ef8798325bdc19d92be428317/templates/Program/program_description.html.twig#L59

Reply:
Removed the repeated div